### PR TITLE
[Feature][Transform-SQL]Support sql transform to generate UUID

### PR DESCRIPTION
**Pull request by zhilinli123**
### Check list

* [ ] If any new Jar binary package adding in your PR, please add License Notice according
  [New License Guide](https://github.com/apache/seatunnel/blob/dev/docs/en/contribution/new-license.md)
* [ ] If necessary, please update the documentation to describe the new feature. https://github.com/apache/seatunnel/tree/dev/docs
* [ ] If you are contributing the connector code, please check that the following files are updated:
  1. Update [plugin-mapping.properties](https://github.com/apache/seatunnel/blob/dev/plugin-mapping.properties) and add new connector information in it
  2. Update the pom file of [seatunnel-dist](https://github.com/apache/seatunnel/blob/dev/seatunnel-dist/pom.xml)
  3. Add ci label in [label-scope-conf](https://github.com/apache/seatunnel/blob/dev/.github/workflows/labeler/label-scope-conf.yml)
  4. Add e2e testcase in [seatunnel-e2e](https://github.com/apache/seatunnel/tree/dev/seatunnel-e2e/seatunnel-connector-v2-e2e/)
  5. Update connector [plugin_config](https://github.com/apache/seatunnel/blob/dev/config/plugin_config)
* [ ] Update the [`release-note`](https://github.com/apache/seatunnel/blob/dev/release-note.md).

### CODE REVIEW

File: seatunnel-e2e/seatunnel-transforms-v2-e2e/seatunnel-transforms-v2-e2e-part-2/src/test/resources/sql_transform/sql_all_columns.conf [hunk 1]
```diff
@@ -45,77 +45,86 @@ transform {
   Sql {
     source_table_name = "fake"
     result_table_name = "fake1"
-    query = "select *, id as id_ from fake"
+    query = "select *, id as id_,UUID() as uuid from fake"
```
Comment (Hisoka-X) [inline seatunnel-e2e/seatunnel-transforms-v2-e2e/seatunnel-transforms-v2-e2e-part-2/src/test/resources/sql_transform/sql_all_columns.conf]: Please put test case into https://github.com/apache/seatunnel/blob/139919334df43b8c320cb1139700a05124f55358/seatunnel-e2e/seatunnel-transforms-v2-e2e/seatunnel-transforms-v2-e2e-part-2/src/test/resources/sql_transform/func_string.conf#L57

**Review by Hisoka-X — COMMENTED** [review]
Thanks @zhilinli123 !

File: seatunnel-e2e/seatunnel-transforms-v2-e2e/seatunnel-transforms-v2-e2e-part-2/src/test/resources/sql_transform/sql_all_columns.conf (hunk 1 above)
Comment (zhilinli123) [reply seatunnel-e2e/seatunnel-transforms-v2-e2e/seatunnel-transforms-v2-e2e-part-2/src/test/resources/sql_transform/sql_all_columns.conf]: done

File: docs/en/transform-v2/sql-functions.md [hunk 2]
```diff
@@ -973,3 +973,14 @@ It is used to determine whether the condition is valid and return different valu
 Example:
 
 case when c_string in ('c_string') then 1 else 0 end
+
+### UUID
+
+```UUID()```
```
Comment (Hisoka-X) [inline docs/en/transform-v2/sql-functions.md:979]:
```suggestion
```UUID()```

Generate a uuid through java function.
```

**Review by Hisoka-X — APPROVED** [review]
LGTM. Thanks @zhilinli123 !

**Comment by hailin0** [comment]
waiting for ci passed

**Review by hailin0 — APPROVED** [review]

**Pull request merged**